eiffel-vocabulary/EiffelActivityCanceledEvent.md

```diff
@@ -182,8 +183,6 @@ __Description:__ A URI pointing at a location from where the schema used when cr
 | 2.0.0 | [edition-agen](../../../tree/edition-agen) | Introduced purl identifiers instead of GAVs (see [Issue 182](https://github.com/eiffel-community/eiffel/issues/182)) |
 | 1.1.0 | [edition-toulouse](../../../tree/edition-toulouse) | Multiple links of type FLOW_CONTEXT allowed. |
 | 1.0.0 | [edition-bordeaux](../../../tree/edition-bordeaux) | Initial version. |
-
-
 ## Examples
```

Comment:
It's unfortunate that we're dropping the blank line between the history table and the following section header. It's a royal pain to get all the newlines right but could we try a bit harder? I wrote the darn thing in the first place so I can take a crack at it.

Reply:
Sick , it's wrong and I need to fix it. It should be 1 newline and not 2 but I have had problems as this only is about whitespaces

Reply:
Fixed for now in [63c8447](https://github.com/eiffel-community/eiffel/pull/304/commits/63c8447a07d2843bc71c2b9b466714e1b17cf96e) with an ugly hack, @magnusbaeck if you have some better suggestion please let me know.

https://ttl255.com/jinja2-tutorial-part-3-whitespace-control/ tells us that adding a `-` in the end will trim all whitespaces not just the line its on and putting in before doesn't help (why?).